### PR TITLE
Fixed some grammatical issues, comments and code segments; added information on mouse and keyboard actions

### PR DESCRIPTION
**Pull request by User253489**
Fixed some grammatical issues, comments and code segments; added information on mouse and keyboard actions

### CODE REVIEW

File: best.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 <p>Functional testing is difficult to get right for many reasons.
  As if application state, complexity, and dependencies don't make testing difficult enough,
- dealing with browsers – and especially cross-browser incompatibilities –
+ dealing with browsers (especially with cross-browser incompatibilities)
```
Comment (andreastt) [inline best.html]: Keep dashes, alternatively use commas.

The dashes are currently used to highlight that this is an important fact, and placing it in parenthesis makes for the opposite effect.

File: best.html [hunk 2]
```diff
@@ -33,34 +33,34 @@
 an interface to a page of your AUT. The tests then use the methods of
 this page object class whenever they need to interact with that page
 of the UI. The benefit is that if the UI changes for the page, the
-tests themselves don’t need to change, only the code within the page
-object needs to change. Subsequently all changes to support that new
+tests themselves don’t need to change; only the code within the page
```
Comment (andreastt) [inline best.html]: Keep comma. A semicolon should be used when listing multiple statements that each contain a comma, and is superfluous in this context.

File: index.html [hunk 3]
```diff
@@ -111,7 +111,7 @@
    <li><a href=wd.html#keyboard>Keyboard</a>
    <li><a href=wd.html#mouse>Mouse</a>
   </ul></li>
- <li><a href=remote.html#remote_webdriver>Remote WebDriver</a>
+ <li><a href=remote.html#remote_webdriver>RemoteWebDriver</a>
```
Comment (andreastt) [inline index.html]: This does not describe the `RemoteWebDriver` interfaces, but the concept of a _remote WebDriver_.

File: java.html [hunk 4]
```diff
@@ -13,21 +13,21 @@
 
 <h2>Working with colours</h2>
 
-<p>You will occasionally want to validate the colour of a thing as part of your tests,
+<p>You will occasionally want to validate the colour of something as part of your tests;
```
Comment (andreastt) [inline java.html:16]: Drop use of semicolon.

File: remote.html [hunk 5]
```diff
@@ -1,18 +1,18 @@
 <!doctype html>
 <meta charset=utf-8>
-<title>Remote WebDriver</title>
+<title>RemoteWebDriver</title>
```
Comment (andreastt) [inline remote.html]: Keep old title.

File: remote.html [hunk 6]
```diff
@@ -1,18 +1,18 @@
...
 <link rel="icon" href="favicon.ico" type="image/vnd.microsoft.icon">
 <link rel="shortcut icon" href="favicon.ico" type="image/vnd.microsoft.icon">
 <link rel=stylesheet href=se.css>
 <link rel=prev href=wd.html title=WebDriver>
 <link rel=next href=best.html title="Best Practices">
 <script src=docs.js></script>
 
-<h1>Remote WebDriver</h1>
+<h1>RemoteWebDriver</h1>
```
Comment (andreastt) [inline remote.html]: Revert.

File: remote.html [hunk 7]
```diff
@@ -1,18 +1,18 @@
...
 
-You can use Remote webdriver the same way you would use webdriver
-locally. The primary difference is that remote webdriver needs to be
-configured so that it can run your tests on a seperate machine.
+You can use RemoteWebDriver the same way you would use WebDriver
```
Comment (andreastt) [inline remote.html]: This should in fact say:

> You can use WebDriver remotely, …

**Review by andreastt — CHANGES_REQUESTED** [review]
Sorry for the late review.

Please make the necessary changes I have highlighted. I would also have appreciated if you made one commit per file changed, but it’s fine to remember this for future PRs.

Please also keep the first line of your commit messages around 50 characters and use imperative language.

**Review by andreastt — APPROVED** [review]

**Comment by User253489** [comment]
Thanks for the review.

I've made some but not all of the changes you highlighted. The ones I didn't make are such that our viewpoints on them differ but you can now make them if you want.

From now on, I'll (1) perform one commit per modified file, and (2) write my commit messages in the manner you requested.

**Comment by andreastt** [comment]
@User253489 As you see, this does not apply to master because changes have been done in the interim. Do you want to have a shot at rebasing it?

**Comment by User253489** [comment]
Thanks for merging the changes after I rebased.

**Pull request merged**